controllers/v1/userProjects.js

```diff
@@ -139,9 +140,6 @@ module.exports = class UserProjects extends Abstract {
                     req.headers["x-app-ver"] : 
                     req.headers.appversion ? req.headers.appversion : ""
                 );
-
```

Comment:
@Prajwal17Tunerlabs why do we require a token ?

Reply:
we need token to communicate with entity service